File: BehsaApp/BehsaApp.Web/Modules/Behsa/Members/MembersForm.cs
using Serenity.ComponentModel;
using System;

namespace BehsaApp.Behsa.Forms;

[FormScript("Behsa.Members")]
[BasedOnRow(typeof(MembersRow), CheckNames = true)]
public class MembersForm
{
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public DateTime BirthDate { get; set; }
    public int? ProvinceId { get; set; }
    public int CityId { get; set; }
}

File: BehsaApp/BehsaApp.Web/Modules/ServerTypes/Behsa/MembersForm.ts
import { StringEditor, DateEditor, LookupEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface MembersForm {
    FirstName: StringEditor;
    LastName: StringEditor;
    BirthDate: DateEditor;
    ProvinceId: LookupEditor;
    CityId: LookupEditor;
}

export class MembersForm extends PrefixedContext {
    static formKey = 'Behsa.Members';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!MembersForm.init)  {
            MembersForm.init = true;

            var w0 = StringEditor;
            var w1 = DateEditor;
            var w2 = LookupEditor;

            initFormType(MembersForm, [
                'FirstName', w0,
                'LastName', w0,
                'BirthDate', w1,
                'ProvinceId', w2,
                'CityId', w2
            ]);
        }
    }
}